### DISCUSSION TITLE
sbt-pekko-version-check sbt plugin?

### DISCUSSION THREAD

**Discussion by mkurz** [apache/pekko#671] [General]
Hi,

in Play we use the [sbt-akka-version-check](https://github.com/johanandren/sbt-akka-version-check) sbt plugin.
Do you think it would make sense if you fork it and adjust it for Pekko? Like as official Pekko plugin/project. Or should we just fork it ourselves?
Currently the plugin is not so relevant, since there is only one major Pekko release right now, but maybe when more major Pekko versions are out it might be useful.

What do you think?

**Comment by pjfanning** [comment]
The ASF can't touch that code unless it is donated to us. I don't think that is going to happen in this case.

**Reply by mkurz** [reply]
Alright, good to know. We might fork then, if we keep using it.

**Reply by pjfanning** [reply]
https://github.com/Philippus/sbt-pekko-version-check now exists
